saml2/saml2-service-provider/src/opensaml4Test/java/org/springframework/security/saml2/provider/service/authentication/OpenSaml4AuthenticationProviderTests.java

```diff
@@ -190,9 +192,7 @@ public void authenticateWhenMissingSubjectThenThrowAuthenticationException() {
 		Response response = response();
 		Assertion assertion = assertion();
 		assertion.setSubject(null);
-		TestOpenSamlObjects.signed(assertion, TestSaml2X509Credentials.assertingPartySigningCredential(),
-				RELYING_PARTY_ENTITY_ID);
-		response.getAssertions().add(assertion);
+		response.getAssertions().add(signed(assertion));
```

Comment:
Thanks for thinking about making the other tests easier. Since they aren't part of the feature you are adding, will you please move these other test changes to a separate commit in this PR?